docs/source/markdown/options/env.update.md

```diff
@@ -0,0 +1,13 @@
+####> This option file is used in:
+####>   podman update
+####> If file is edited, make sure the changes
+####> are applicable to all of those.
+#### **--env**, **-e**=*env*
+
+Add a value (e.g. env=*value*) to the built image. Can be used multiple times.
+If the value already exists in the built image, it is overridden.
+To remove an environment variable from the image, use the `--unsetenv`
+option.
```

Comment:
this needs a s/image/container/ as update operates on the container and no longer cares of the base env from the image.